packages/medusa/src/services/line-item.js

```diff
@@ -89,17 +89,22 @@ class LineItemService extends BaseService {
     return lineItem
   }
 
-  async generate(variantId, regionId, quantity, metadata = {}) {
+  async generate(variantId, regionId, quantity, metadata = {}, unitPrice) {
```

Comment:
Should we maybe make the last argument here `config` so we have:

`async generate(variantId, regionId, quantity, config = {})`

`config` would then be able to include: `{ metadata: {}, unit_price: [num] }` potentially more stuff down the line